src/domain/documentation/DocumentationPage.tsx

```
@@ -8,14 +9,68 @@ import Gutters from '@/core/ui/grid/Gutters';
+      }
+
+      switch (event.data.type) {
+        case SupportedMessageTypes.PageHeight:
+          if (event.data.height) {
+            setHeight(`${event.data.height}px`);
+          }
+          break;
+        case SupportedMessageTypes.PageChange:
+          // on page change just replace the path (used for sharing)
+          // the actual navigation is internal for the iframe
+          const newPath = `/${TopLevelRoutePath.Docs}${event.data.url}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)